### ISSUE TITLE
Update Quick Start Guide - VirtualBox is installed by Vagrant

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/drupal-vm#357]
Installation is much simpler as of Vagrant 1.8.0—it will install VirtualBox automatically if it's not already present, so the Quick Start Guide can literally be updated to:

  1. Install Vagrant.
  2. Download or clone this project.
  3. `vagrant up`

Especially considering #355 (using the new `ansible_local` provisioner), Vagrant 1.8.0 has made things oh-so-much more amazing :)

**Comment by ashabed** [comment]
Due to a bug (?) in the latest version of ansible, the ansible_local provisioner does not yet seem to play nice with DrupalVM. I think you're aware.

A student from debugacademy.com's drupalvm instance stopped working after running vagrant destroy, because the next vagrant up updated the packages from the project. I worked with them to update vagrant & virtualbox, but it turned out we needed to install ansible locally to get drupalvm working again. I ran vagrant destroy & vagrant up on my computer and found the exact same thing.

It may be worth adding installing ansible as a requirement in the quick start instructions until that issue is sorted out.

**Comment by geerlingguy** [comment]
See #372 — until Vagrant 1.8.2 is released (why oh why do they wait?!), I recommend anyone without Ansible installed on the host stick with Drupal VM 2.1.2 or patch Vagrant to use the fix mentioned in #372. It's already been merged into Vagrant's dev branch, we're just waiting on a release.

Note that everything worked fine until Ansible 2.0 was released; it changed an exit flag Vagrat was using, so that's what broke the new local provisioner.
